integration/container_interface.go

```diff
@@ -22,16 +22,8 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
-type ContainerSpec struct {
-	Name         string
-	Cmd          string
-	Options      []containerOption
-	Cleanup      bool
-	StartAndStop bool
-}
-
 type ContainerFactory interface {
-	NewContainer(ContainerSpec) ContainerInterface
+	NewContainer(name, cmd string, opts ...containerOption) ContainerInterface
 }
 
 type ContainerInterface interface {
```

Comment:
```suggestion
type ContainerInterface interface {
	testutils.Container
	IsCleanup() bool
	IsStartAndStop() bool
}
```
Do you think this makes sense? We don't need some methods of `testutils.Container` but they way we created them they already have it.
Or maybe even combining the interfaces alltogether and make them one

Reply:
I don't know either what to do :D
I just got confused while reviewing `type ContainerInterface interface` and `type Container interface` and that was just a quick thought

Reply:
I renamed it to `IntegrationTestsContainer`.